EIPS/eip-6865.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 6865
+title: on-chain EIP-712 Visualization
```

Comment:
```suggestion
title: On-Chain EIP-712 Visualization
```

Reply:
Fixed in commit [573ff5b1ebf](https://github.com/ethereum/EIPs/pull/6865/commits/573ff5b1ebfb3329d5e9b404d5ef8b47c78a58f3)